docs/notes/2.31.x.md

```diff
@@ -49,7 +49,7 @@ used by Pants itself works.
 
 Added the `[python].default_to_resolve_interpreter_constraints` option which when set the true, allows Python targets with both `interpreter_constraints` and `resolve` fields to fallback on their resolve's interpreter constraints before the global constraints if no interpreter constraints are set on the actual target.
 
-The version of [Pex](https://github.com/pex-tool/pex) used by the Python backend has been upgraded to [`v2.73.1`](https://github.com/pex-tool/pex/releases/tag/v2.73.1). This fixes two issues:
+The version of [Pex](https://github.com/pex-tool/pex) used by the Python backend has been upgraded to [`v2.76.1`](https://github.com/pex-tool/pex/releases/tag/v2.76.1). This fixes two issues:
```

Comment:
In spite of 2.73.1 being the update that was referenced by the subsequent two bugs, I've just updated this here - as updating to 2.76.1 should have the same effect.